root/collection/execTClonesArrayAbsorb.C

```diff
@@ -1,9 +1,6 @@
 void execTClonesArrayAbsorb() {
-#ifndef ClingWorkAroundMissingImplicitAuto
-   TClonesArray *tc, *tc2;
-#endif
-tc = new TClonesArray("TNamed",5); for(int i=0; i<20; ++i) tc->ConstructedAt(i);
-tc2 = new TClonesArray("TNamed",5); for(int i=0; i<20; ++i) tc2->ConstructedAt(i);
+TClonesArray *tc = new TClonesArray("TNamed",5); for(int i=0; i<20; ++i) tc->ConstructedAt(i);
```

Comment:
> The test is supposed to test leaks in TClonesArray, not implicit auto. 

Do we have a test or JIRA ticket that capture **this** use case?  (I.e. when we work on fixing implicit auto, are we reducing the test coverage by jettison this use case?)